{"@t":"2020-03-02T11:41:29.7041317Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s); to create a disconnected multiplexer, disable AbortOnConnectFail. SocketFailure on PING\r\n   at StackExchange.Redis.ConnectionMultiplexer.ConnectImpl(Func`1 multiplexerFactory, TextWriter log) in c:\\code\\StackExchange.Redis\\StackExchange.Redis\\StackExchange\\Redis\\ConnectionMultiplexer.cs:line 889\r\n   at StackExchange.Redis.ConnectionMultiplexer.Connect(String configuration, TextWriter log) in c:\\code\\StackExchange.Redis\\StackExchange.Redis\\StackExchange\\Redis\\ConnectionMultiplexer.cs:line 855\r\n   at Microsoft.Extensions.Caching.Redis.RedisCache.Connect()\r\n   at Microsoft.Extensions.Caching.Redis.RedisCache.GetAndRefresh(String key, Boolean getData)\r\n   at Microsoft.Extensions.Caching.Redis.RedisCache.Get(String key)\r\n   at Sell.Core.Caching.RedisCacheManager.IsSet(String key) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Core\\Caching\\RedisCacheManager.cs:line 47\r\n   at Sell.Core.Caching.Extensions.GetAsych[T](ICacheManager cacheManager, String key, Int32 cacheTime, Func`1 GetFromDb) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Core\\Caching\\Extensions.cs:line 24\r\n   at Sell.Services.Catalog.Product.ProductService.SearchProductByFilterAsync(ProductFilterDTO productFilterDTO) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Product\\ProductService.cs:line 130\r\n   at SellAPI.Controllers.ProductController.FindByFilter(ProductFilterDTO productFilterDTO) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\ProductController.cs:line 43\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTUJQ0U91Q8:00000001","RequestPath":"/api/Product/findbyfilter","SpanId":"|784d7439-4f227319ee31dc7b.","TraceId":"784d7439-4f227319ee31dc7b","ParentId":"","ConnectionId":"0HLTUJQ0U91Q8"}
{"@t":"2020-03-03T10:30:49.5307344Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 39\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 41\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVBNHOGLAB:00000001","RequestPath":"/api/Category/","SpanId":"|9b03fa78-4a0bc82ea0ffeb86.","TraceId":"9b03fa78-4a0bc82ea0ffeb86","ParentId":"","ConnectionId":"0HLTVBNHOGLAB"}
{"@t":"2020-03-03T10:32:44.9289350Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 39\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 41\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVBO639RNC:00000001","RequestPath":"/api/Category/","SpanId":"|c8ed6f50-44ca6ce57c937cc0.","TraceId":"c8ed6f50-44ca6ce57c937cc0","ParentId":"","ConnectionId":"0HLTVBO639RNC"}
{"@t":"2020-03-03T10:33:47.7667935Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 39\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 41\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVBP7LGUU2:00000001","RequestPath":"/api/Category/","SpanId":"|f7efae2d-4e279c700798a86f.","TraceId":"f7efae2d-4e279c700798a86f","ParentId":"","ConnectionId":"0HLTVBP7LGUU2"}
{"@t":"2020-03-03T10:45:12.6852678Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 39\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 42\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVBVN5T5FE:00000001","RequestPath":"/api/Category/","SpanId":"|bb40156-4fa8dcdbeacfbe55.","TraceId":"bb40156-4fa8dcdbeacfbe55","ParentId":"","ConnectionId":"0HLTVBVN5T5FE"}
{"@t":"2020-03-03T10:48:06.1937841Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 39\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 42\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC13VP0ER:00000001","RequestPath":"/api/Category/","SpanId":"|da00f573-47b514f74a7ebfb9.","TraceId":"da00f573-47b514f74a7ebfb9","ParentId":"","ConnectionId":"0HLTVC13VP0ER"}
{"@t":"2020-03-03T10:49:30.1182318Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 39\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 42\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC1PEBB6E:00000001","RequestPath":"/api/Category/","SpanId":"|dec80854-44776034ad8a13d1.","TraceId":"dec80854-44776034ad8a13d1","ParentId":"","ConnectionId":"0HLTVC1PEBB6E"}
{"@t":"2020-03-03T10:51:41.5363271Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 48\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC2UASDG6:00000001","RequestPath":"/api/Category/","SpanId":"|727ed2d0-4ffe283111bc632c.","TraceId":"727ed2d0-4ffe283111bc632c","ParentId":"","ConnectionId":"0HLTVC2UASDG6"}
{"@t":"2020-03-03T10:55:07.4960939Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 48\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC542L1VT:00000001","RequestPath":"/api/Category/","SpanId":"|5c3d478c-4f21d8b4f20af2cb.","TraceId":"5c3d478c-4f21d8b4f20af2cb","ParentId":"","ConnectionId":"0HLTVC542L1VT"}
{"@t":"2020-03-03T10:56:16.9688141Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 49\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC5TVTKFM:00000001","RequestPath":"/api/Category/","SpanId":"|859a123c-43a8f06a69b6e631.","TraceId":"859a123c-43a8f06a69b6e631","ParentId":"","ConnectionId":"0HLTVC5TVTKFM"}
{"@t":"2020-03-03T10:58:56.6166838Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 49\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC7CUGQIP:00000001","RequestPath":"/api/Category/","SpanId":"|7467923-4ec3a9d390444d20.","TraceId":"7467923-4ec3a9d390444d20","ParentId":"","ConnectionId":"0HLTVC7CUGQIP"}
{"@t":"2020-03-03T10:58:58.4690327Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 49\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC7CUGQIP:00000002","RequestPath":"/api/Category/","SpanId":"|7467924-4ec3a9d390444d20.","TraceId":"7467924-4ec3a9d390444d20","ParentId":"","ConnectionId":"0HLTVC7CUGQIP"}
{"@t":"2020-03-03T10:58:59.6403702Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 49\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC7CUGQIP:00000003","RequestPath":"/api/Category/","SpanId":"|7467925-4ec3a9d390444d20.","TraceId":"7467925-4ec3a9d390444d20","ParentId":"","ConnectionId":"0HLTVC7CUGQIP"}
{"@t":"2020-03-03T10:59:34.0432645Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 50\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC7OD2RVU:00000001","RequestPath":"/api/Category/","SpanId":"|c939a25a-4e5225fb4c8bb776.","TraceId":"c939a25a-4e5225fb4c8bb776","ParentId":"","ConnectionId":"0HLTVC7OD2RVU"}
{"@t":"2020-03-03T11:01:07.9414727Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 50\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVC83COLTH:00000001","RequestPath":"/api/Category/","SpanId":"|39ce4c4f-40969838effa0ad6.","TraceId":"39ce4c4f-40969838effa0ad6","ParentId":"","ConnectionId":"0HLTVC83COLTH"}
{"@t":"2020-03-03T11:05:40.1844638Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidCastException: The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.ValidateType(Object value)\r\n   at Microsoft.Data.SqlClient.SqlParameterCollection.Add(Object value)\r\n   at Sell.Data.SQLServerApplicationContext.RunSp[T](String StoreName, List`1 ListParamert) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\SQLServerApplicationContext.cs:line 41\r\n   at Sell.Data.Repositores.EFRepository`1.RunFunctionDb[T](String functionName, List`1 paramters) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 134\r\n   at Sell.Services.Catalog.Category.CategoryService.SearchAllCategoryAsync() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Category\\CategoryService.cs:line 50\r\n   at SellAPI.Controllers.CategoryController.GetAll() in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\CategoryController.cs:line 27\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVCB1C7U4S:00000001","RequestPath":"/api/Category/","SpanId":"|218563f4-49eae48a54dcd885.","TraceId":"218563f4-49eae48a54dcd885","ParentId":"","ConnectionId":"0HLTVCB1C7U4S"}
{"@t":"2020-03-03T11:58:40.8395020Z","@m":"Failed executing DbCommand (\"89\"ms) [Parameters=[\"@p0='?' (DbType = DateTime2), @p1='?' (Size = 4000), @p2='?' (DbType = Int32), @p3='?' (DbType = DateTime2), @p4='?' (Size = 4000)\"], CommandType='Text', CommandTimeout='30']\"\r\n\"\"SET NOCOUNT ON;\r\nINSERT INTO [Picture] ([CreateOn], [MimeType], [ProductID], [UpdateOn], [VirtualPath])\r\nVALUES (@p0, @p1, @p2, @p3, @p4);\r\nSELECT [ID]\r\nFROM [Picture]\r\nWHERE @@ROWCOUNT = 1 AND [ID] = scope_identity();\"","@i":"627a98df","@l":"Error","elapsed":"89","parameters":"@p0='?' (DbType = DateTime2), @p1='?' (Size = 4000), @p2='?' (DbType = Int32), @p3='?' (DbType = DateTime2), @p4='?' (Size = 4000)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [Picture] ([CreateOn], [MimeType], [ProductID], [UpdateOn], [VirtualPath])\r\nVALUES (@p0, @p1, @p2, @p3, @p4);\r\nSELECT [ID]\r\nFROM [Picture]\r\nWHERE @@ROWCOUNT = 1 AND [ID] = scope_identity();","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","ActionId":"9ae18cce-997a-4642-9b88-929d76b0ca53","ActionName":"SellAPI.Controllers.PictureController.UploadAsync (SellAPI)","RequestId":"0HLTVD5TS12U0:00000001","RequestPath":"/api/Picture","SpanId":"|7b7199bd-44b13267c57ae6a5.","TraceId":"7b7199bd-44b13267c57ae6a5","ParentId":"","ConnectionId":"0HLTVD5TS12U0"}
{"@t":"2020-03-03T11:58:41.2753750Z","@m":"An exception occurred in the database while saving changes for context type '\"Sell.Data.SQLServerApplicationContext\"'.\"\r\n\"\"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint \\\"FK_Picture_Product_ProductID\\\". The conflict occurred in database \\\"SellAPI\\\", table \\\"dbo.Product\\\", column 'ID'.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:66424dc8-c8e3-481c-99cf-5aedf17d0381\r\nError Number:547,State:0,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\"","@i":"170f029e","@l":"Error","@x":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_Picture_Product_ProductID\". The conflict occurred in database \"SellAPI\", table \"dbo.Product\", column 'ID'.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:66424dc8-c8e3-481c-99cf-5aedf17d0381\r\nError Number:547,State:0,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","contextType":"Sell.Data.SQLServerApplicationContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_Picture_Product_ProductID\". The conflict occurred in database \"SellAPI\", table \"dbo.Product\", column 'ID'.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:66424dc8-c8e3-481c-99cf-5aedf17d0381\r\nError Number:547,State:0,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","ActionId":"9ae18cce-997a-4642-9b88-929d76b0ca53","ActionName":"SellAPI.Controllers.PictureController.UploadAsync (SellAPI)","RequestId":"0HLTVD5TS12U0:00000001","RequestPath":"/api/Picture","SpanId":"|7b7199bd-44b13267c57ae6a5.","TraceId":"7b7199bd-44b13267c57ae6a5","ParentId":"","ConnectionId":"0HLTVD5TS12U0"}
{"@t":"2020-03-03T11:58:41.9120570Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint \"FK_Picture_Product_ProductID\". The conflict occurred in database \"SellAPI\", table \"dbo.Product\", column 'ID'.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:66424dc8-c8e3-481c-99cf-5aedf17d0381\r\nError Number:547,State:0,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Sell.Data.Repositores.EFRepository`1.InsertAsync(TEntity entity) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Data\\Repositores\\EFRepository.cs:line 110\r\n   at Sell.Services.Catalog.Picture.PictureService.RegisterPictureAsync(PictureUploadDTO pictureUploadDTO) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\Sell.Services\\Catalog\\Picture\\PictureService.cs:line 27\r\n   at SellAPI.Controllers.PictureController.UploadAsync(PictureUploadDTO image) in C:\\Users\\AliHF\\Source\\Repos\\SellAPI\\SellAPI\\Controllers\\PictureController.cs:line 46\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVD5TS12U0:00000001","RequestPath":"/api/Picture","SpanId":"|7b7199bd-44b13267c57ae6a5.","TraceId":"7b7199bd-44b13267c57ae6a5","ParentId":"","ConnectionId":"0HLTVD5TS12U0"}
{"@t":"2020-03-03T13:56:30.6590924Z","@m":"Exception occurred while processing message.","@i":"d37e9c4d","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.IO.IOException: IDX20804: Unable to retrieve document from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)\r\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectConfigurationRetriever.GetAsync(String address, IDocumentRetriever retriever, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()","EventId":{"Id":3,"Name":"ProcessingMessageFailed"},"SourceContext":"Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler","RequestId":"0HLTVFAGKMR60:00000001","RequestPath":"/api/Product/","SpanId":"|8ae53534-4e75f406ece1b262.","TraceId":"8ae53534-4e75f406ece1b262","ParentId":"","ConnectionId":"0HLTVFAGKMR60"}
{"@t":"2020-03-03T13:56:30.7293689Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.IO.IOException: IDX20804: Unable to retrieve document from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)\r\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectConfigurationRetriever.GetAsync(String address, IDocumentRetriever retriever, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.AuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationService.AuthenticateAsync(HttpContext context, String scheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVFAGKMR60:00000001","RequestPath":"/api/Product/","SpanId":"|8ae53534-4e75f406ece1b262.","TraceId":"8ae53534-4e75f406ece1b262","ParentId":"","ConnectionId":"0HLTVFAGKMR60"}
{"@t":"2020-03-03T22:56:31.9408654Z","@m":"Exception occurred while processing message.","@i":"d37e9c4d","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.IO.IOException: IDX20804: Unable to retrieve document from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)\r\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectConfigurationRetriever.GetAsync(String address, IDocumentRetriever retriever, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()","EventId":{"Id":3,"Name":"ProcessingMessageFailed"},"SourceContext":"Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler","RequestId":"0HLTVOODJUGQC:00000001","RequestPath":"/api/Product/","SpanId":"|43cc4444-4da273dde8998fde.","TraceId":"43cc4444-4da273dde8998fde","ParentId":"","ConnectionId":"0HLTVOODJUGQC"}
{"@t":"2020-03-03T22:56:32.1010352Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.IO.IOException: IDX20804: Unable to retrieve document from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.\r\n ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)\r\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)\r\n   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.HttpDocumentRetriever.GetDocumentAsync(String address, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectConfigurationRetriever.GetAsync(String address, IDocumentRetriever retriever, CancellationToken cancel)\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.AuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationService.AuthenticateAsync(HttpContext context, String scheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.<Invoke>g__AwaitMatcher|8_0(EndpointRoutingMiddleware middleware, HttpContext httpContext, Task`1 matcherTask)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVOODJUGQC:00000001","RequestPath":"/api/Product/","SpanId":"|43cc4444-4da273dde8998fde.","TraceId":"43cc4444-4da273dde8998fde","ParentId":"","ConnectionId":"0HLTVOODJUGQC"}
{"@t":"2020-03-03T22:56:36.6337428Z","@m":"Exception occurred while processing message.","@i":"d37e9c4d","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()","EventId":{"Id":3,"Name":"ProcessingMessageFailed"},"SourceContext":"Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler","RequestId":"0HLTVOODJUGQC:00000002","RequestPath":"/api/Product/","SpanId":"|43cc4445-4da273dde8998fde.","TraceId":"43cc4445-4da273dde8998fde","ParentId":"","ConnectionId":"0HLTVOODJUGQC"}
{"@t":"2020-03-03T22:56:36.6387678Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.AuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationService.AuthenticateAsync(HttpContext context, String scheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVOODJUGQC:00000002","RequestPath":"/api/Product/","SpanId":"|43cc4445-4da273dde8998fde.","TraceId":"43cc4445-4da273dde8998fde","ParentId":"","ConnectionId":"0HLTVOODJUGQC"}
{"@t":"2020-03-03T22:56:40.1943273Z","@m":"Exception occurred while processing message.","@i":"d37e9c4d","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()","EventId":{"Id":3,"Name":"ProcessingMessageFailed"},"SourceContext":"Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler","RequestId":"0HLTVOODJUGQC:00000003","RequestPath":"/api/Product/","SpanId":"|43cc4446-4da273dde8998fde.","TraceId":"43cc4446-4da273dde8998fde","ParentId":"","ConnectionId":"0HLTVOODJUGQC"}
{"@t":"2020-03-03T22:56:40.2045806Z","@m":"An unhandled exception has occurred while executing the request.","@i":"48a46595","@l":"Error","@x":"System.InvalidOperationException: IDX20803: Unable to obtain configuration from: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.\r\n   at Microsoft.IdentityModel.Protocols.ConfigurationManager`1.GetConfigurationAsync(CancellationToken cancel)\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationHandler`1.AuthenticateAsync()\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationService.AuthenticateAsync(HttpContext context, String scheme)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)","EventId":{"Id":1,"Name":"UnhandledException"},"SourceContext":"Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware","RequestId":"0HLTVOODJUGQC:00000003","RequestPath":"/api/Product/","SpanId":"|43cc4446-4da273dde8998fde.","TraceId":"43cc4446-4da273dde8998fde","ParentId":"","ConnectionId":"0HLTVOODJUGQC"}
